getting_started/macos_apple_silicon.md

```diff
@@ -40,11 +40,7 @@ you need to install one or more of these platform language compilers, too.
     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
     ```
 
-1. Install the Zig compiler, for apps with Zig-based platforms:
-
-    ```sh
-    brew install zig@0.9.1
-    ```
```

Comment:
zig 0.9.1 is no longer available on homebrew

Reply:
Might be worth explicitly mentioning that, otherwise some people might decide to use homebrew instead of following the instructions.

Reply:
Good point, I'll set up a new PR